src/HttpClient/Client.php

```diff
@@ -46,189 +55,84 @@ public function __construct(?Guzzle $client = null, ?LoggerInterface $logger = n
      */
     public function request(string $method, string $uri, ?array $options = null): ResponseInterface
     {
-        $this->logRequest($method, $uri, $options);
-
-        // Define exception in case request fails
-        $exception = null;
-
-        try {
-            $request = $this->client->request($method, $uri, $options ?? []);
-            $content = $this->getBodyContents($request->getBody());
-
-            $response = new Response(
-                $this->processResponseContent($content),
-                $request->getStatusCode(),
-                $request->getHeaders(),
-                $content
-            );
-        } catch (RequestException $exception) {
-            $response = $this->handleRequestException($exception);
-        } catch (GuzzleException $exception) {
-            // Covers any other guzzle exception
-            $response = new Response(['content' => $exception->getMessage()], 500);
+        $request = new Request(
+            $method,
+            $uri,
+            $options['headers'] ?? [],
```

Comment:
How come not having version as a method arg?

Reply:
It comes from Guzzle, but I dont think Guzzle supports 2.0?